### ISSUE TITLE
sasview not working with neutron problem definitions

### ISSUE THREAD

**Issue by Anders-Markvardsen** [fitbenchmarking/fitbenchmarking#206]
Sasview integration with #199 provides working solution for all combination minimizer/problem definition files, expect this remaining combination.

The reproduce this bug: in example_runScripts.py set
```
software = ['sasview']
problem_sets = ["Neutron"]
```
which results in the error
```
  File "C:\Midlertiddig\fitbenchmarking\fitbenchmarking\fitting\sasview\main.py", line 109, in fit
    formatted_param_names = [param[0] for param in problem.starting_values]
TypeError: 'NoneType' object is not iterable
```

Add tests for the above.

Also, with #199 Coverage decreased (-9.2%) to 82.931%. In this issue or in separate issue increase Coverage back to (82.931+9.2)% ensuring that there is tests for all combinations of minimizers and types of problem definition files

**Comment by AtomAnu** [comment]
@Anders-Markvardsen I believe I know the answer to this. I will assign myself to this task.

**Comment by Anders-Markvardsen** [comment]
thanks

**Comment by tyronerees** [comment]
Fixed by PR #263
